packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -56,57 +52,85 @@ const getDefaultComponents = (
     const additionalCharCount =
```

Comment:
Do all languages separate lists by commas?

Reply:
Good point. I think Arabic does not for 3 or less. This will take me more time in a separate ticket and instead of strings we may need to keep a callback in the locale file instead.